lib/versioning/git/index.ts

```diff
@@ -1,26 +1,31 @@
-import * as generic from '../loose/generic';
+import { regEx } from '../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../loose/generic';
 import type { VersioningApi } from '../types';
 
 export const id = 'git';
 export const displayName = 'git';
 export const urls = ['https://git-scm.com/'];
 export const supportsRanges = false;
 
-const parse = (version: string): any => ({ release: [parseInt(version, 10)] });
```

Comment:
Should `{ release: [NaN] }` still be considered valid?

Reply:
This is what I think as well. However, I wasn't sure if we have to restrict hex strings by length 40 or 8, so I've made valid any hex sequence including the empty one.

Reply:
@rarkins WDYT?